.release/buildspec_stage.yml

```diff
@@ -22,7 +22,7 @@ phases:
       - |
         for artifact in `cat ./$GIT_COMMIT_ID.manifest`
         do
-            md5FileName="${artifact%.exe}"
```

Comment:
im so confused - we used to have ".exe" at the end of all md5 files? but that's clearly not the case here: https://github.com/aws/copilot-cli/releases what am I misunderstanding?